.github/workflows/build-artifacts.yml

```diff
@@ -16,21 +18,24 @@ jobs:
         uses: actions/setup-dotnet@v1
         with:
           dotnet-version: '3.1.x'
+          source-url: https://nuget.pkg.github.com/graphql-dotnet/index.json
+        env:
+          NUGET_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Where does it come from?

Reply:
The GitHub token has predefined permissions and is a built in feature. One of the permissions is publishing a package to GitHub packages for its own registry. It has an advantage of being invalid once the workflow ends.

Reply:
Do I need to configure/enable this token from GitHub UI ?

Reply:
No it's always enabled

Reply:
https://docs.github.com/en/actions/reference/authentication-in-a-workflow

https://github.com/actions/setup-dotnet